integrations/oci/sdk/cdi/src/main/java/io/helidon/integrations/oci/sdk/cdi/OciExtension.java

```diff
@@ -652,14 +665,17 @@ private void processInjectionPoint(@Observes ProcessInjectionPoint<?, ?> event)
     }
 
     private void afterBeanDiscovery(@Observes AfterBeanDiscovery event, BeanManager bm) {
-        for (ServiceTaqs serviceTaqs : this.serviceTaqs) {
-            if (serviceTaqs.isEmpty()) {
-                installAdps(event, bm, serviceTaqs.qualifiers());
-            } else {
-                TypeAndQualifiers serviceClientBuilder = serviceTaqs.serviceClientBuilder();
-                TypeAndQualifiers serviceClient = serviceTaqs.serviceClient();
-                installServiceClientBuilder(event, bm, serviceClientBuilder, serviceClient, this.lenientClassloading);
-                installServiceClient(event, bm, serviceClient, serviceTaqs.serviceInterface(), serviceClientBuilder);
+        if (!this.serviceTaqs.isEmpty()) {
```

Comment:
Can we add a test to ensure we are falling back to microprofile config?

Reply:
Like:
```
// in a unit test somewhere
io.helidon.config.Config c = toHelidonConfig(ConfigProvider.getConfig());
OciRuntimeExtension.fallbackConfigSupplier(() -> c);
assertThat(OciRuntimeExtension.configSupplier().get(), is(c));
```
…? Or did you have something else in mind?

Reply:
Yes, put something into the microprofile-config.properties file, and ensure that it got picked up by your code (which will internally defer to OciExtension, etc.)

Reply:
OK…I can do this, but isn't this testing the `io.helidon.integrations.oci.sdk.runtime.OciExtension` class? Are you sure you want the test of how that class should behave in _this_ Java module? There's no place in my code that could regress in this area (I just hand you a converted MicroProfile Config `Config` object and you go from there).

Reply:
I added a test that hopefully gets at what you're looking for; not sure.